### PR TITLE
Subway Stops problem

### PR DESCRIPTION
**Pull request by antleo1995**
Created a function to calculate the number of stops - sorry it's late.

### CODE REVIEW

File: lib/mbta.js [hunk 1]
```diff
@@ -1,11 +1,24 @@
...
+  'Red': ['South Station', 'Park Street', 'Kendall', 'Central', 'Harvard',
+    'Porter', 'Davis', 'Alewife'],
+  'Green': ['Government Center', 'Park Street', 'Boylston', 'Arlington',
+    'Copley', 'Hynes', 'Kenmore'],
+  'Orange': ['North Station', 'Haymarket', 'Park Street', 'State',
+    'Downtown Crossing', 'Chinatown', 'Back Bay', 'Forest Hills']
+
+}
 
 const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
```
Comment (benjimelito) [inline lib/mbta.js:14]: Arrow function ftw

File: lib/mbta.js [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+  let startingIndex = subLines[startLine].indexOf(startStation)
+  let endingIndex = subLines[endLine].indexOf(endStation)
+  let numOfStartingStops = Math.abs(startingIndex - subLines[startLine].indexOf('Park Street'))
+  let numOfEndigStops = Math.abs(endingIndex - subLines[endLine].indexOf('Park Street'))
```
Comment (benjimelito) [inline lib/mbta.js:18]: Careful - the riders don't always pass through Park Street on their journey. Some sort of check to see if they are switching lines would be helpful here.

**Comment by benjimelito** [comment]
The code looks 💵 , I think the only thing you're missing is a bit of the logic.

File: lib/mbta.js (hunk 2 above)
Comment (antleo1995) [reply lib/mbta.js:18]: Yeah, I see you point. It works fine as best I can tell for all cases where park street is passed but if they don't pass park street then i am certain my logic fails. I will revisit this with fresh eyes and some further training.

File: lib/mbta.js (hunk 1 above)
Comment (antleo1995) [reply lib/mbta.js:14]: I wasn't sure if I missed this in class, but just kind of assumed this was shorthand for a function. Javascript loves shorthand I see.

**Pull request closed without merging**